spec/yaml/swiftnav/sbp/observation.yaml

```diff
@@ -1110,3 +1110,122 @@ definitions:
       - isc_l2c:
           type: s16
           units: s * 2^-35
+
+ - AlmanacCommonContent:
+    short_desc: Common fields for every almanac message.
+    fields:
+        - sid:
+            type: GnssSignal
+            desc: GNSS signal identifier
+        - toa:
+            type: GPSTime
+            desc: Reference time of almanac
+        - ura:
+            type: double
+            units: m
+            desc: User Range Accuracy
+        - fit_interval:
+            type: u32
+            units: s
+            desc: Curve fit interval
+        - valid:
+            type: u8
+            desc: Status of ephemeris, 1 = valid, 0 = invalid
+        - health_bits:
+            type: u8
+            desc: |
+              Satellite health status.
+              GPS: ICD-GPS-200, chapter 20.3.3.3.1.4
+              SBAS: 0 = valid, non-zero = invalid
+              GLO: 0 = valid, non-zero = invalid
+
+ - MSG_ALMANAC_GPS:
+    id: 0x0070
+    short_desc: Satellite broadcast ephemeris for GPS
+    desc: |
+      The almanac message returns a set of satellite orbit parameters. Almanac
+      data is not very precise and is considered valid for up to several months.
+      Please see the Navstar GPS Space Segment/Navigation user interfaces
+      (ICD-GPS-200, Chapter 20.3.3.5.1.2 Almanac Data) for more details.
+    fields:
+      - common:
+          type: AlmanacCommonContent
+          desc: Values common for all almanac types
+      - m0:
+          type: double
+          units: rad
+          desc: Mean anomaly at reference time
+      - ecc:
+          type: double
+          desc: Eccentricity of satellite orbit
+      - sqrta:
+          type: double
+          units: m^(1/2)
+          desc: Square root of the semi-major axis of orbit
+      - omega0:
+          type: double
+          units: rad
+          desc: Longitude of ascending node of orbit plane at weekly epoch
+      - omegadot:
+          type: double
+          units: rad/s
+          desc: Rate of right ascension
+      - w:
+          type: double
+          units: rad
+          desc: Argument of perigee
+      - inc:
+          type: double
+          units: rad
+          desc: Inclination
+      - af0:
+          type: double
+          units: s
+          desc: Polynomial clock correction coefficient (clock bias)
+      - af1:
+          type: double
+          units: s/s
+          desc: Polynomial clock correction coefficient (clock drift)
```

Comment:
IS-GPS-200H Table 20-VI. Units are correct and scale factors are 2^(-20) and 2^(-38).